types/attestation.yaml

```diff
@@ -37,25 +37,8 @@ PendingAttestation:
       $ref: "./primitive.yaml#/BitList"
       description: "Attester aggregation bits."
     data:
-      $ref: '#/AttestationData'
+      $ref: './attestation_data.yaml#/AttestationData'
     inclusion_delay:
       $ref: "./primitive.yaml#/Uint64"
     proposer_index:
-      $ref: "./primitive.yaml#/Uint64"
-
-AttestationData:
-  type: object
-  description: "The [`AttestationData`](https://github.com/ethereum/consensus-specs/blob/v1.3.0/specs/phase0/beacon-chain.md#attestationdata) object from the CL spec."
-  required: [slot, index, beacon_block_root, source, target]
-  properties:
-    slot:
-      $ref: "./primitive.yaml#/Uint64"
-    index:
-      $ref: "./primitive.yaml#/Uint64"
-    beacon_block_root:
-      $ref: './primitive.yaml#/Root'
-      description: "LMD GHOST vote."
-    source:
-      $ref: "./misc.yaml#/Checkpoint"
-    target:
-      $ref: "./misc.yaml#/Checkpoint"
+      $ref: "./primitive.yaml#/Uint64"
```

Comment:
missing new line here causes strange diff in line 44

Reply:
Actually I moved the `AttestationData` to a separate schema file since it's shared by both Attestation and `Electra.Attestation`. No need to add a new line then